### PR TITLE
Fixes #707 pkgdown: "standard" theme

### LINKED ISSUE #707: pkgdown: "standard" theme
Let's set the theme of the online documentation to "standard" (dark text, white background)

Also some parts of the vignettes in the online documentation are not displayed correctly. 
Not sure if it's because of the theme or other settings

E.g. s. https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/articles/introduction-re-instances.html#nomenclature

grafik

### PR DESCRIPTION
**Pull request by Yuri05**
`pkgdown::build_site() `produced some warnings, partucularly some images are not displayed correctly. probably problem with some vignettes ?
But in general looks good, the tables which were not properly renedered in the past seems to be ok now

```
2: Missing images in 'vignettes/introduction-re-instances.Rmd': '../../../Users/XXXXX/AppData/Local/Temp/RtmpCuFoPE/temp_libpath66902e256b0a/ospsuite.reportingengine/extdata/mean-model-workflow-input-and-tasks.png', '../../../Users/XXXXX/AppData/Local/Temp/RtmpCuFoPE/temp_libpath66902e256b0a/ospsuite.reportingengine/extdata/population-workflow-input-and-tasks.png'
i pkgdown can only use images in 'man/figures' and 'vignettes' 
3: The vignette title specified in \VignetteIndexEntry{} is different from the title in the YAML metadata. The former is "pk-parameters", and the latter is "PK parameters". If that is intentional, you may set options(rmarkdown.html_vignette.check_title = FALSE) to suppress this check. 
4: The vignette title specified in \VignetteIndexEntry{} is different from the title in the YAML metadata. The former is "plot-settings", and the latter is "Plot settings". If that is intentional, you may set options(rmarkdown.html_vignette.check_title = FALSE) to suppress this check. 
5: Missing images in 'vignettes/sensitivity-analysis.Rmd': '../../../Users/XXXXX/AppData/Local/Temp/RtmpCuFoPE/temp_libpath66902e256b0a/ospsuite.reportingengine/extdata/sa-file-outputs.png', '../../../Users/XXXXX/AppData/Local/Temp/RtmpCuFoPE/temp_libpath66902e256b0a/ospsuite.reportingengine/extdata/sa-file-index.png'
i pkgdown can only use images in 'man/figures' and 'vignettes' 
```

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite.reportingengine
 Title: R package to create reports for the Open Systems Pharmacology models
-Version: 2.0.0
+Version: 2.0.0.9000
```
Comment (Yuri05) [inline DESCRIPTION:4]: switched to the "pkgdown developer version"

File: _pkgdown.yml [hunk 2]
```diff
@@ -2,11 +2,6 @@ url: https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/
...
-  bootswatch: cyborg
-  theme: arrow-dark
-  bslib:
-    primary: "#0054AD"
-    border-radius: 0.5rem
-    btn-border-radius: 0.25rem
-
 
+development:
+  mode: auto
```
Comment (Yuri05) [inline _pkgdown.yml:7]: removed dark theme and added developemnt auto mode https://pkgdown.r-lib.org/reference/build_site.html#development-mode

The site was built then under docs/dev.
All other changes are the built site

File: docs/dev/pkgdown.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+pandoc: 2.9.2.1
```
Comment (IndrajeetPatil) [inline docs/dev/pkgdown.yml:1]: You don't need to make this change now, but I would recommend using a more recent version of `pandoc`.

You can install it from here:
https://pandoc.org/installing.html

**Review by IndrajeetPatil — APPROVED** [review]
Looks good to me!

The warnings you see are related to the relative paths being specified in vignettes. I think @pchelle knows how to fix them.

The other warning about title not matching the vignette entry can be solved easily.

The entry here:

https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/e20c05c6cb8cafc0f38ef5c5de99c28e1242bdc3/vignettes/pk-parameters.Rmd#L2

should match what's here:

https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/e20c05c6cb8cafc0f38ef5c5de99c28e1242bdc3/vignettes/pk-parameters.Rmd#L10

Assuming this is what you want, both of them should be `PK parameters`.

**Comment by Yuri05** [comment]
Seems that the package tidyselect cannot be downloaded from CRAN. Ideas?

```
trying URL 'http://cran.us.r-project.org/bin/windows/contrib/4.0/tidyselect_1.1.2.zip'
Error in download.file(url, destfile, method, mode = "wb", ...) : 
  cannot open URL 'http://cran.us.r-project.org/bin/windows/contrib/4.0/tidyselect_1.1.2.zip'
In addition: Warning message:
In download.file(url, destfile, method, mode = "wb", ...) :
  cannot open URL 'http://cran.us.r-project.org/bin/windows/contrib/4.0/tidyselect_1.1.2.zip': HTTP status was '404 Not Found'
Warning in download.packages(pkgs, destdir = tmpd, available = available,  :
  download of package 'tidyselect' failed
```

**Comment by IndrajeetPatil** [comment]
You will have to install it from source because binaries are still not built for it. Its new release came out only yesterday, so it will take a few days for them to be built.

For now, this might work:

```yaml
- Rscript -e "install.packages(c('ospsuite.utils','dplyr','purrr','covr','readr','tidyr'), repos = 'http://cran.us.r-project.org', type='win.binary')"
- Rscript -e "install.packages('tidyselect', repos = 'http://cran.us.r-project.org', type='source')"
```

**Comment by Yuri05** [comment]
yep, this helped. package is created now.
Some tests fail, but this has probably nothing to do with the current PR.
Would suggest to merge the PR and create another issue for the tests

```
== Failed tests ================================================================
  -- Failure (test-study-design.R:67:3): Units and paths are checked and converted to base unit when loading a study design --
  `addStudyParameters(testPopulation, testSimulation, studyDesignTempFile)` produced warnings.
  -- Failure (test-study-design.R:105:3): Units and paths are checked and converted to base unit when loading a study design --
  `addStudyParameters(testPopulation, testSimulation, studyDesignTempFile)` produced warnings.
  -- Failure (test-study-design.R:141:3): A study design 'SOURCE' requires a 'MIN', 'MAX', or 'EQUALS' attribute --
  `addStudyParameters(testPopulation, testSimulation, studyDesignMinFile)` produced warnings.
  -- Failure (test-study-design.R:142:3): A study design 'SOURCE' requires a 'MIN', 'MAX', or 'EQUALS' attribute --
  `addStudyParameters(testPopulation, testSimulation, studyDesignMaxFile)` produced warnings.
  -- Failure (test-study-design.R:143:3): A study design 'SOURCE' requires a 'MIN', 'MAX', or 'EQUALS' attribute --
  `addStudyParameters(testPopulation, testSimulation, studyDesignEqualsFile)` produced warnings.
  
  [ FAIL 5 | WARN 97 | SKIP 0 | PASS 351 ]
```

**Comment by pchelle** [comment]
> The warnings you see are related to the relative paths being specified in vignettes. I think @pchelle knows how to fix them.

- I had created a `figures` folder within the `vignette` folder and dumped all png images within.
- Then, in the yaml header of the vignette, I had set this folder for export (code below)
```yaml
resource_files:
  - figures
```
- Finally, I had referenced the relative path using `figures/...`

**Review by pchelle — APPROVED** [review]
I can take care of the images warning in a different issue if needed

**Comment by Yuri05** [comment]
Seemed to work with the DEV-documentation: https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/dev/

**Pull request merged**